client/app/components/browse-list/template.hbs

```diff
@@ -4,10 +4,10 @@
             {{#if list}}
                 <div class="col-xs-6">
                     {{#if organizeMode}}
-                        {{#unless groupView}}
-                            <button class="btn btn-default" {{action 'toggleGroupConfirmation'}}>Group Selected</button>
-                        {{/unless}}
-                        <button class="btn btn-danger" {{action 'toggleDeleteConfirmation'}}>Delete selected </button>
+                        {{#if (not groupView)}}
+                            <button class="btn btn-default {{unless selectedItems.length 'disabled'}}" {{action 'toggleGroupConfirmation'}}>Group Selected</button>
+                        {{/if}}
+                        <button class="btn btn-danger {{unless selectedItems.length 'disabled'}}" {{action 'toggleDeleteConfirmation'}}>Delete selected </button>
```

Comment:
The delete button appears to be disabled, but is still clickable:

![screen shot 2017-04-07 at 9 54 45 am](https://cloud.githubusercontent.com/assets/6414394/24803135/44157cbc-1b78-11e7-8622-8d73d960ea33.png)